web/modules/custom/portland/modules/portland_media_embed_helper/portland_media_embed_helper.module

```diff
@@ -87,8 +87,12 @@ function portland_media_embed_helper_form_entity_embed_dialog_alter(array &$form
     if (isset($form['attributes']['data-align'])) unset($form['attributes']['data-align']);
   }
 
-  // Hide additional fields from entity_embed module
-  $form['attributes']['data-entity-embed-display-settings']['#access'] = false;
+  // Allow view mode selection, move to bottom of dialog, and disable ajax since we don't have additional settings to load
+  $form['attributes']['data-entity-embed-display']['#access'] = TRUE;
+  $form['attributes']['data-entity-embed-display']['#weight'] = 99;
+  unset($form['attributes']['data-entity-embed-display']['#ajax']);
+  // Hide additional unused fields from entity_embed module
+  $form['attributes']['data-entity-embed-display-settings']['#access'] = FALSE;
```

Comment:
It's seems to be sufficient to only set `$form['attributes']['data-entity-embed-display-settings']` which could simplify the code a little. See the PR link in the previous comment.